common/platforms/sky130osu12Ths/config.mk

```diff
@@ -0,0 +1,111 @@
+# cell prefix
+export cell_prefix = sky130_osu_sc_12T_hs
```

Comment:
can you use 12t instead of 12T (everywhere else). The capital is standing out ..

Reply:
Certainly!

Reply:
I just realized that it comes like that from pdk.. please dismiss this change.